src/utils/ramdaHelpers.ts

```diff
@@ -0,0 +1,9 @@
+import * as R from 'ramda';
+
+type GenericObj = { [k: string]: any };
```

Comment:
Let's put generic type declarations in a single file somewhere and import them. Also this is better known as `Dictionary<any>`.